pkg/sql/sessiondatapb/local_only_session_data.proto

```diff
@@ -97,8 +97,6 @@ message LocalOnlySessionData {
   // ImplicitPartitioningEnabled indicates whether implicit column partitioning
   // can be created.
   bool implicit_column_partitioning_enabled = 22;
-  // DropEnumValueEnabled indicates whether enum values can be dropped.
-  bool drop_enum_value_enabled = 23;
```

Comment:
Shouldn't this be replaced with `reserved 23` until 22.1 is released?

Reply:
nice catch, i wasn't familiar with the right way to do this